nvflare/lighter/templates/master_template.yml

```diff
@@ -275,6 +275,8 @@ fed_server: |
                 "target": "{~~target~~}",
                 "scheme": "{~~scheme~~}"
             },
+            "admin_server": "{~~admin_server~~}",
```

Comment:
where are we using this "admin_server"?

Reply:
To be backward compatible with the preflight tool. Flare system does not need this.